infra/pr_helper.py

```diff
@@ -75,7 +75,7 @@ def main():
   pr_author = github.get_pr_author()
   # Gets all modified projects path.
   projects_path = github.get_projects_path()
-  verified, email = github.get_author_email()
+  email = github.get_author_email()
```

Comment:
What's the reason for this change? Are we still verifying that the email is valid?

Reply:
We removed the verified check in the previous PR (https://github.com/google/oss-fuzz/pull/10519), so I removed this variable.  It seems that most emails are not verified, which is confusing for users (https://github.com/google/oss-fuzz/pull/10509#issuecomment-1588862254). If we don't want to remove the verified check, I can move the contact check after the PR commit history check.

Reply:
Ah I missed that part in my last review. I think as is this would also break because `github.get_author_email()` still returns a tuple? 

Can we add that verified check back? @jonathanmetzman any reasons for removing this? Otherwise this is trivial for contributors to spoof. We can simplify the output message here to just add a "(verified)" string if needed.

Reply:
Ah I misunderstood this. Let's keep it. Sorry, Holly.

Reply:
No problem, added email verification back and simplified message.